package.json

```diff
@@ -9,6 +9,7 @@
     "dist": "npm run dist-dev && npm run dist-prod",
     "dist-dev": "browserify -s Kinto -d -e src/index.js -o demo/kinto.dev.js",
     "dist-prod": "browserify -s Kinto -g uglifyify -e src/index.js -o demo/kinto.min.js",
+    "dist-fx": "browserify -s loadKinto -d -e fx-src/index.js -o dist/temp.jsm && cp fx-src/jsm_prefix.js dist/moz-kinto-client.js && cat dist/temp.jsm | grep -v sourceMappingURL >> dist/moz-kinto-client.js && rm dist/temp.jsm",
```

Comment:
Several notes here:

- The `-d` option exports source maps; remove it and you don't need to `grep -v` them anymore;
- The `dist` folder has been removed along #217, so we shouldn't use it; or at least making sure to create the folder — or a temporary one — first;
- I can't where does the `jsm_prefix.jsm` file come from; hints? Shouldn't it be `temp.js`?